style.css

```diff
@@ -1 +1,357 @@
+@import /*  Add your styling here */
+  url('https://fonts.googleapis.com/css2?family=Cookie&family=Montserrat&display=swap');
 /*  Add your styling here */
+:root {
+  --green: #15f1a7;
+  --yellow: #edf50c;
+  --green-light: #8be5f3;
+  --red-light: #ffcab0;
+  --blue-light: #83cee0;
+  --purple: #ea6af6;
+  --pink: #ef61cb;
+  --black: #000;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  background-color: var(--green-light);
+  height: 100vh;
+  font-family: "Montserrat", sans-serif;
+}
+/* header */
+header {
+  grid-area: 1 / 1 / 2 / 2;
+  background-color: var(pink);
+  /* header grid */
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  height: 200px;
+  margin-bottom: 0;
+}
+
+.logo {
+  grid-area: 1 / 1 / 3 / 2;
+  background-image: #8be5f3;
+}
+.logo img {
+  width: 200px;
+  height: 200px;
+}
+
+header p {
+  display: flex;
+  justify-content: center;
+  font-family: "Cookie", sans-serif;
+  font-size: 20px;
+  margin: 10px;
+}
+/* header navigation */
+nav {
+  grid-area: 1 / 2 / 2 / 3;
+}
+
+header ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+header ul li {
+  margin: 10px;
+}
+
+/* menu */
+.navigation {
+  display: none;
+}
+.menu {
+  border: 1px solid var(--green);
+  padding: 1rem;
+  position: fixed;
+  z-index: 2;
+  background-color: var(--red-light);
+  opacity: 0.9;
+  top: -800px;
+  right: 70px;
+  transition: all 0.3s ease-in-out;
+  border-radius: 5px;
+}
+header ul li a {
+  text-decoration: none;
+  color: var(--black);
+}
+
+#check {
+  display: none;
+}
+
+/* burger menu */
+.open-menu {
+  top: 5%;
+  right: 20px;
+  transform: translateY(-50%);
+  position: absolute;
+  cursor: pointer;
+  transition: all 0.4s ease;
+}
+
+#check:checked ~ .open-menu {
+  transform: rotate(-90deg);
+}
+#check:checked ~ .menu {
+  top: 0;
+}
+/* header text */
+header p {
+  grid-area: 2 / 2 / 3 / 3;
+}
+
+/* main */
+main {
+  grid-area: 2 / 1 / 3 / 2;
+  /* main grid */
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  background-color: var(--pink);
+  height: fit-content;
+}
+.container {
+  grid-area: 1 / 1 / 2 / 2;
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.container h1 {
+  font-size: 42px;
+  font-family: "Cookie", sans-serif;
+}
+
+.container p {
+  font-size: 30px;
+}
+
+.mainpicture {
+  grid-area: 2 / 1 / 3 / 2;
+  background-image: url(/image/Anniversary-Cake-Birthday-60th-Diamond-1024x731.jpeg);
```

Comment:
This is probably a content image, so should be a `img` element in the HTML. 
`background-image` is good for decorative images like icons.